src/app/config/confparse.c

```diff
@@ -642,7 +642,7 @@ config_dump(const config_format_t *fmt, const void *default_options,
       continue;
     }
     /* Don't save 'hidden' control variables. */
-    if (!strcmpstart(fmt->vars[i].member.name, "__"))
+    if (fmt->vars[i].flags & CVFLAG_NODUMP)
```

Comment:
Should we have a function to check this flag?

Reply:
Okay!  Done in 078ba326a1255dbfbb442219c599e7961f4f85e2